docs/layouts/sub-navigation.md

```diff
@@ -58,3 +58,9 @@ related:
   summaryText: "Common front matter options",
   html: detailsText
 }) }}
+
+In addition to the common front matter options, this layout also accepts the following options:
+
+| Name                 | Type   | Description                                                                                                                                                |
+| :------------------- | :----- | :--------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **subNavigationKey** | string | Parent navigation key (usually a page title) to show items below in the sub navigation. Default is `homeKey` value provided in [plugin options](/options). |
```

Comment:
It felt slightly confusing to me that this was called `subNavigationKey` but described as "parent navigation key"... Took me a min to get my head around what it was doing.  

I wonder if `sectionKey` or `subSectionKey` or similar would work?

Reply:
Updated to use `sectionKey`